content.go

```diff
@@ -98,3 +100,22 @@ func Resolve(ctx context.Context, target Target, ref string, opts ResolveOptions
 
 	return selectPlatform(ctx, target, desc, opts.TargetPlatform)
 }
+
+var ErrInvalidMediaType error = errors.New("invalid Media Type")
+
+// GenerateDescriptor returns an OCI descriptor, given the content and media type.
+func GenerateDescriptor(content []byte, mediaType string) (ocispec.Descriptor, error) {
+	if mediaType == "" {
+		return ocispec.Descriptor{}, ErrInvalidMediaType
+	}
+	return ocispec.Descriptor{
+		MediaType: mediaType,
+		Digest:    digest.FromBytes(content),
+		Size:      int64(len(content)),
+	}, nil
+}
+
+// Equal tests if two OCI descriptors are identical.
```

Comment:
Maybe: 
> Equal returns true if two OCI descriptors point to the same content.

With this exposed, the internal `EqualOCI` method is no longer needed:
https://github.com/oras-project/oras-go/blob/main/internal/descriptor/descriptor.go#L81-L84

Reply:
Do I need to delete the internal method?

Reply:
Yes, as they are duplicate.

Reply:
Changed the doc comment and deleted the internal method.